src/Fennel/CSharp.fs

```diff
@@ -100,6 +109,7 @@ type Prometheus =
         
     static member Comment(text) = P.comment text
     static member Help(name, text) = P.help name text
+    static member MetricType(name, t) = P.typeHint name (t |> MetricTypeEnumHelpers.ToMetricType)
```

Comment:
Based on the above suggestions this would become `member this.MetricType = metricType |> MetricTypeEnum.FromMetricType`